apis/r/configure

```diff
@@ -10,8 +10,14 @@ if test -z "${R_HOME}"; then
 fi
 
 ### for single-cell-data/apis/r repo, consider completing sources
-if test -f src/lib/libtiledbsoma.a; then
-    echo "** static library present"
+pkg-config --exists tiledbsoma
+if [ $? -eq 0 ]; then
+    lib=`pkg-config --libs-only-L tiledbsoma` | cut -d'_' -f 2
+    export LD_LIBRARY_PATH=$lib:$LD_LIBRARY_PATH
+    sudo ldconfig
```

Comment:
Notwithstanding the previous comment, we should not need the change to LD_LIBRARY_PATH, or the call to `ldconfig`.